Bio/SearchIO/_legacy/NCBIStandalone.py

```diff
@@ -1305,7 +1305,7 @@ def align(self, line):
         if len(seq) < self._query_len:
             # Make sure the alignment is the same length as the query
             seq += " " * (self._query_len - len(seq))
-        elif len(seq) < self._query_len:
+        if len(seq) < self._query_len:
```

Comment:
Can be part of the previous `if` block

Reply:
True - indented this would be a little clearer.